modules/jaggery-apps/appmgt/src/site/pages/createapplication.jag

```diff
@@ -22,6 +22,10 @@ var site = require("/site/conf/site.json");
 
 (function () {
     var type = request.getParameter('type');
+    var apptype = request.getParameter('apptype');
+    var apptypeIconColor = request.getParameter('apptypeIconColor');
+    var apptypeIconImageClass = request.getParameter('apptypeIconImageClass');
+    var appTypeDisplayName = request.getParameter('appTypeDisplayName');

```

Comment:
What is the rationale behind getting these information from appType bean?. Note that this is a 'page' and all the pages follow the request.getParameter way. Also I don't see any harm of using this way, particularly for this parameters come as POST not as url params.